util/libssl.num

```diff
@@ -564,12 +564,12 @@ SSL_use_sign_PrivateKey                 614	1_1_1e	EXIST::FUNCTION:NTLS,SM2
 SSL_use_enc_certificate                 615	1_1_1e	EXIST::FUNCTION:NTLS,SM2
 SSL_use_enc_certificate_file            616	1_1_1e	EXIST::FUNCTION:NTLS,SM2
 SSL_use_enc_PrivateKey                  617	1_1_1e	EXIST::FUNCTION:NTLS,SM2
-SSL_CTX_get_options2                    620	1_1_1g	EXIST::FUNCTION:OPTIONS2
```

Comment:
Please delete useless code.

Reply:
Don't quite understand this...

Reply:
No options2 API, so no NOEXIST::FUNCTION: